src/types/DataTypes.tsx

```diff
@@ -139,7 +139,7 @@ export interface LocaleData extends ObjectBase {
   languageCode: LanguageCode;
   territoryCode: TerritoryCode;
   explicitScriptCode?: ScriptCode;
-  variantTag?: VariantIANATag;
+  variantTag?: VariantTagData; // variantIANAtag to variantTagData
```

Comment:
Split this. We should keep the raw string as well as the object
```suggestion
  variantTagCode?: VariantIANATag
  variantTagData?: VariantTagData;
```